PytHW1/Task1.py

```diff
@@ -0,0 +1,10 @@
+print('Введите число')
+a = int(input())
+sum = 0
+i = -1 
+
+while i <= len(str(a)):
+    sum = sum + a%10
+    a //= 10
+    i+=1
```

Comment:
Без переменной `i` можно обойтись. Да и зачем считать длину строкового представления переменной `а`?
```suggestion
while a:
    sum += a % 10
    a //= 10
```